asciidoc/get-started.adoc

```diff
@@ -46,15 +46,16 @@ See also the <<http-api#http-api, HTTP API documentation>>.
 == Driver versions and installation
 
 Starting with Neo4j 4.0, the versioning scheme for the database, driver and protocol have all been aligned.
-This simplifies general compatibility concerns and reduces mental overhead.
+This simplifies general compatibility concerns.
 
-That said, cross-version compatibility is still available, and we guarantee minimum support for current and previous versions between both server and driver.
-Concretely, this means that Neo4j 4.0 is guaranteed compatible with both 4.0 and 1.7 series drivers, and the 4.0 series drivers are guaranteed compatible with both Neo4j 4.0 and Neo4j 3.5.
+That said, cross-version compatibility is still available, and minimum support for current and previous versions between both server and driver is guaranteed.
+More specifically, this means that Neo4j 4.0 is guaranteed to be compatible with both 4.0 and 1.7 drivers, and the 4.0 drivers are guaranteed to be compatible with both Neo4j 4.0 and Neo4j 3.5.
 In cases where at least one peer is below version 4.0, communication will occur in fallback mode, limiting functionality to that available in the lowest-versioned component.
 
 Wherever possible, it is recommended to use the latest driver release available.
 This will provide the greatest degree of stability and will ensure that the full set of server functionality is available.
-The drivers make no explicit distinction between editions, however, and simply operate with the functionality made available by Neo4j itself.
+The drivers, when used with Neo4j Enterprise Edition, come with full cluster routing support.
+The drivers make no explicit distinction between Enterprise Edition and Community Edition, however, and simply operate with the functionality made available by Neo4j itself.
```

Comment:
I'd suggest:
The drivers make no explicit distinction between Enterprise Edition and Community Edition however, and simply operate with the functionality made available by Neo4j itself.
(Removed comma after Community Edition)

Reply:
My suggestion is very minor, and otherwise I agree with all your changes @mariascharin .